lib/gridfs-stream/upload.js

```diff
@@ -447,3 +486,15 @@ function writeRemnant(_this, callback) {
     checkDone(_this);
   });
 }
+
+/**
+ * @ignore
+ */
+
+function checkAborted(_this, callback) {
```

Comment:
I think that call is there: https://github.com/mongodb/node-mongodb-native/blob/5d9b6d5c23d5e1a491939f3f4bc2aca4c7c0f6a9/lib/gridfs-stream/upload.js#L499